src/main/resources/application.yaml

```
@@ -0,0 +1,8 @@
+server:
+  port: 8081
+
+spring:
+  redis:
+    host:
+    port:
+    password:
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)